bundle/language_en.properties

```diff
@@ -664,19 +664,24 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.acanceledinventorycannotbemodified.msg                                                 = A canceled inventory cannot be modified.
 angal.inventory.allproduct.radio                                                                       = All products
-angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg                          = Cannot create another inventory with other in progress in this ward.
+angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg                          = Cannot create another inventory with another in progress in this ward.
 angal.inventory.date.label                                                                             = Inventory's date
-angal.inventory.edit.title                                                                             = Edit an inventory
-angal.inventory.error.msg                                                                              = Error during  some products's recording
+angal.inventory.edit.title                                                                             = Edit An Inventory
```

Comment:
```suggestion
angal.inventory.edit.title                                                                             = Edit a Inventory
```

Reply:
Actually `an` is grammatically correct.     `an` is used when the next work starts with a vowel; otherwise `a` is used.

Reply:
And why not just `Edit Inventory`?   We don't say `Edit a Patient` do we?